modules/targetid/test/hist_test.cpp

```diff
@@ -0,0 +1,63 @@
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE TargetIdentification
+#include "histogram.h"
+#include <boost/test/unit_test.hpp>
+#include <boost/log/trivial.hpp>
+#include <iostream>
+#include <fstream>
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include "target_identifier.h"
+#include "canny_contour_creator.h"
+#include "k_means_filter.h"
+#include "frame.h"
+#include "benchmark.h"
+#include "contour_comparison.h"
+using namespace std;
+using namespace cv;
+using namespace boost;
+
+BOOST_AUTO_TEST_CASE(hist_test){
+    vector<int> n;
+    n.push_back(3);
+    string telemetry_path=boost::unit_test::framework::master_test_suite().argv[2];
+    string filePath=boost::unit_test::framework::master_test_suite().argv[1];
+    ofstream fout(telemetry_path);
```

Comment:
There are a couple of things I don't like about outputting this csv.
First of all It's not actually necessary for a test that's got nothing to do with the metadata.
But also you're putting it in a fixed-path source directory (and then of course the generated file gets included in the commit), if anything it should go with the generated build files. I would point out however that we do have the actual telemetry logs for the photos in the testdata directory. It would make more sense to include them and use them instead of generating placeholder files.